src/main/scala/dpla/ingestion3/confs/Ingestion3Conf.scala

```diff
@@ -15,18 +25,16 @@ class Ingestion3Conf(confFilePath: String, providerName: String) extends ConfUti
 
     val baseConfig = ConfigFactory.load
 
-    val providerConf = ConfigFactory
-      .load
-      .getConfig(providerName)
-      .withFallback(baseConfig)
-      .resolve()
+    val providerConf = providerName match {
+      case Some(name) => ConfigFactory.load
+        .getConfig(name)
+        .withFallback(baseConfig)
```

Comment:
With this fallback to `baseConfig` - is this just an empty config, or are there default values?  And if there are default values, where are they defined?

Reply:
`baseConfig` includes all the non-provider specific settings (`spark`, `twofishess` etc.) defined within the application configuration. 

Properties set outside of a provider context can be overridden within a provider block so those are the "defaults" in some sense but the absolute fallback (currently) is accessing the configs via `getOrElse` in the Entry/driver classes. For example: 

`.setMaster(i3Conf.spark.sparkMaster.getOrElse("local[*]"))`

Reply:
Because the `required` properties are different for each operation it didn't make sense to me to do any "generic" validation of the config file. Rather push that validation down into the Entry point classes. 

The twofishes host is only required for the enrichment jobs and even then only if you are running Spatial so I didn't want to throw errors if it wasn't included and we tried to run a harvest job.

Reply:
:+1: